src/lib/permission-exclusions-keto-tuples.ts

```diff
@@ -82,40 +74,40 @@ class KetoTuples {
 
     // Construct patch delta
     // eslint-disable-next-line max-len
-    const patchDeltaArr = this._generatePermissionExclusionComboPatchDeltaArray(addPermissionCombos, deletePermissionCombos)
-    if (patchDeltaArr.length > 0) {
+    const relationshipPatch = this._generatePermissionExclusionComboPatchDeltaArray(addPermissionCombos, deletePermissionCombos)
+    if (relationshipPatch.length > 0) {
       logger.info('Patching permission exclusions in Ory Keto....')
-      await this.oryKetoWriteApi.patchRelationTuples(patchDeltaArr)
+      await this.adminRelationshipApi.patchRelationships({ relationshipPatch })
     }
   }
 
   _generatePermissionExclusionComboPatchDeltaArray (
     addPermissionCombos: string[], deletePermissionCombos: string[]
-  ): PatchDelta[] {
-    let patchDeltaArray: PatchDelta[] = []
+  ): keto.RelationshipPatch[] {
+    let patchDeltaArray: keto.RelationshipPatch[] = []
     patchDeltaArray = patchDeltaArray.concat(addPermissionCombos.map(permissionCombo => {
       const permissionArr = permissionCombo.split(':')
       return {
-        action: 'insert',
+        action: keto.RelationshipPatchActionEnum.Insert,
         relation_tuple: this._generatePermissionTuple(permissionArr[0], permissionArr[1])
       }
     }))
     patchDeltaArray = patchDeltaArray.concat(deletePermissionCombos.map(permissionCombo => {
       const permissionArr = permissionCombo.split(':')
       return {
-        action: 'delete',
+        action: keto.RelationshipPatchActionEnum.Delete,
         relation_tuple: this._generatePermissionTuple(permissionArr[0], permissionArr[1])
       }
     }))
     return patchDeltaArray
   }
 
-  _generatePermissionTuple (permissionX: string, permissionY: string): InternalRelationTuple {
+  _generatePermissionTuple (permissionX: string, permissionY: string): keto.Relationship {
     return {
-      namespace: 'permission',
+      namespace: KETO_NAMESPACES.permission,
       object: permissionY,
-      relation: 'excludes',
-      subject: 'permission:' + permissionX + '#granted'
+      relation: KETO_RELATIONS.excludes,
+      subject_id: `permission:${permissionX}#granted`
```

Comment:
I think we can use subject_set for this.
Instead of `permission:PERMISSIONID#granted`, we can use the following
```
    "subject_set": {
        "namespace": "permission",
        "object": "PERMISSIONID",
        "relation": "granted"
    }
```

Or if we don't need `granted` then it can be plain `subject_id: PERMISSIONID`

Reply:
fixed